### PR TITLE
Option to manage own certs

### PR DESCRIPTION
**Pull request by meyerbro**
We already have other formula that manages our certs.

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -57,3 +57,5 @@ filebeat:
       # default is salt://filebeat/files/ca.pem
       # set to empty to disable
       ssl_cert: salt://mycustom/filebeat/logstash-forwarder.crt
+      # If you want to manage your own certs, set below to False
+      managed_cert = False
```
Comment (gravyboat) [inline pillar.example]: The format here is incorrect, it should be `managed_cert: False`

**Review by gravyboat — CHANGES_REQUESTED** [review]
Looks good minus the one note I made.

File: pillar.example (hunk 1 above)
Comment (meyerbro) [reply pillar.example]: Fixed. :)

**Comment by gravyboat** [comment]
Thanks!

**Pull request merged**